VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -8,6 +8,19 @@
 
 import Foundation
 
+private typealias ResponseDictionaryClosure = (responseDictionary: VimeoClient.ResponseDictionary?) -> Void
+
+private protocol Cache
```

Comment:
Possibly, that's probably outside the scope of this ticket though. I just quickly added this protocol (already outside of the scope) since I noticed the caches we're sharing the same function names.